cmd/main.go

```diff
@@ -94,8 +94,15 @@ func main() {
 		}
 		viper.SetDefault("USER_DATA_FOLDER", path.Join(home, ".meshery", "config"))
 	}
+	_, er := os.Stat(viper.GetString("USER_DATA_FOLDER"))
+	if os.IsNotExist(er) {
+		logrus.Infof("Creating directory for storing user data in %v", viper.GetString("USER_DATA_FOLDER"))
+		errDir := os.MkdirAll(viper.GetString("USER_DATA_FOLDER"), 0755)
```

Comment:
mkdirall would suffice, I don't think we have to stat before calling this.

Reply:
For running locally, i made the check

Reply:
Yeah I guess I can do that directly with Mkdirall